lib/repl.js

```diff
@@ -223,6 +227,28 @@ module.paths = CJSModule._nodeModulePaths(module.filename);
 const writer = (obj) => inspect(obj, writer.options);
 writer.options = { ...inspect.defaultOptions, showProxy: true };
 
+// Converts static import statement to dynamic import statement
+const toDynamicImport = (codeLine) => {
+  let dynamicImportStatement = '';
+  const ast = acornParse(codeLine, { __proto__: null, sourceType: 'module', ecmaVersion: 'latest' });
+  acornWalk.ancestor(ast, {
+    ImportDeclaration(node) {
+      const awaitDynamicImport = `await import(${JSONStringify(node.source.value)});`;
+      if (node.specifiers.length === 0) {
+        dynamicImportStatement += awaitDynamicImport;
+      } else if (node.specifiers.length === 1 && node.specifiers[0].type === 'ImportNamespaceSpecifier') {
+        dynamicImportStatement += `const ${node.specifier[0].local.name} = ${awaitDynamicImport}`;
```

Comment:
seems like there could maybe be an `undefined` in here?
```suggestion
      } else if (node.specifiers.length === 1 && node.specifiers[0] && node.specifiers[0].type === 'ImportNamespaceSpecifier') {
        dynamicImportStatement += `const ${node.specifier[0].local.name} = ${awaitDynamicImport}`;
```

Reply:
What input would result in an `undefined` specifier?

Reply:
```suggestion
      } else if (node.specifiers.length === 1 && node.specifiers[0].type === 'ImportNamespaceSpecifier') {
        dynamicImportStatement += `const ${node.specifiers[0].local.name} = ${awaitDynamicImport}`;
```

Reply:
aha, good catch :-p